src/Tasks/Geocode.js

```diff
@@ -30,8 +30,15 @@ export var Geocode = Task.extend({
   },
 
   initialize: function (options) {
-    options = options || {};
-    options.url = options.url || WorldGeocodingServiceUrl;
+    if (typeof options !== 'undefined') {
```

Comment:
this is bogus because people can pass options and still be using the World Geocoding Service.